server/service/integration_enterprise_test.go

```diff
@@ -54,9 +54,10 @@ func (s *integrationEnterpriseTestSuite) SetupSuite() {
 		License: &fleet.LicenseInfo{
 			Tier: fleet.TierPremium,
 		},
-		Pool:   s.redisPool,
-		Lq:     s.lq,
-		Logger: log.NewLogfmtLogger(os.Stdout),
+		Pool:           s.redisPool,
+		Lq:             s.lq,
+		Logger:         log.NewLogfmtLogger(os.Stdout),
+		EnableCachedDS: true,
```

Comment:
Should we also set this to non-enterprise tests too?

Reply:
no argument against! can do if you think it would be useful

Reply:
Am thinking that using cached MySQL is closer to what we use in production, right?

Reply:
definitely! let me give it a try